pymongo/mongo_client.py

```diff
@@ -919,13 +919,15 @@ def address(self):
         .. versionadded:: 3.0
         """
         topology_type = self._topology._description.topology_type
-        if topology_type == TOPOLOGY_TYPE.Sharded:
+        if topology_type == TOPOLOGY_TYPE.Sharded and \
+                len(self.topology_description.server_descriptions()) > 1:
```

Comment:
We avoid using `\` to split lines. Instead wrap the expression in `()`.

> The preferred way of wrapping long lines is by using Python's implied line continuation inside parentheses, brackets and braces. Long lines can be broken over multiple lines by wrapping expressions in parentheses. These should be used in preference to using a backslash for line continuation.

https://www.python.org/dev/peps/pep-0008/